src/main/java/springboot/model/Book.java

```diff
@@ -87,10 +87,18 @@ public void setCoverImage(String coverImage) {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o) {
```

Comment:
remove equals, hashCode, toString, get and set methods
Use Lombok annotations instead. Add `@Table `